components/Ripple/src/MDCRippleView.m

```diff
@@ -155,6 +155,9 @@ - (void)beginRippleTouchDownAtPoint:(CGPoint)point
   rippleLayer.rippleLayerDelegate = self;
   rippleLayer.fillColor = self.rippleColor.CGColor;
   rippleLayer.frame = self.bounds;
+  if (self.rippleStyle == MDCRippleStyleUnbounded) {
```

Comment:
is this the only change needed? I recall in InkView it being much more elaborate. If it is, this is really great!

Reply:
`MDCInkView` holds a reference to the inkLayer and uses that within _legacy ink_, instead of creating a new layer every time. Additionally since it holds a reference to the layer it updates the `maxRippleRadius` if the style changes. Since Ripple creates a new layer every time this isn't needed.

Reply:
Awesome! thanks for checking